tools/manifest/sourcefile.py

```diff
@@ -451,11 +451,18 @@ def variant_nodes(self):
     @cached_property
     def test_variants(self):
         rv = []
-        for element in self.variant_nodes:
-            if "content" in element.attrib:
-                variant = element.attrib["content"]
-                assert variant == "" or variant[0] in ["#", "?"]
-                rv.append(variant)
+        if self.ext == ".js":
+            for (key, value) in self.script_metadata:
+                if key == b"variant":
+                    rv.append(value.decode("utf-8"))
```

Comment:
I see why this makes sense, but please at least file a (good first) issue for supporting multiple meta lines for other cases where we accept multiple values.